feat(forms): add `Iterate.Count` along with `Iterate.useCount` (#3492)
dnbexperience/eufemia: packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/Iterate/Count.mdx

---
title: 'Count'

> description: '`Iterate.Count` is a helper component / function to return the count of an data array or object.'

Reviewer: ```suggestion
description: '`Iterate.Count` is a helper component / function that returns the count of a data array or object.'
```
